dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeConfig.java

```diff
@@ -438,6 +447,73 @@ private void processSubjectDN(String subjectDN) throws AcmeCaException {
 			}
 		}
 	}
+	
+	/**
+	 * Set the amount of time before certificate expiration to renew the certificate
+	 * 
+	 * @param retries
+	 *            The number of time to try to update a challenge.
+	 */
+	@Trivial
+	private void setRenewBeforeExpirationMs(Long ms) {
+		autoRenew = true;
+		if (ms != null) {
+			if (ms <= 0) { // disable auto renew
```

Comment:
Ok. here it is. Probably should document in metatype.

Reply:
added to the metatype.